tests/_benderjs/ckeditor/static/extensions.js

```diff
@@ -135,6 +135,40 @@
 		assert.areSame( expected, bender.tools.compatHtml( actual, false, true, false, true, true ), message );
 	};
 
+	/**
+	 * Asserts that two objects are deep equal.
+	 *
+	 * @param {Object} expected
+	 * @param {Object} actual
+	 * @param {String} [message]
+	 */
+	bender.objectAssert.areDeepEqual = function areDeepEqual( expected, actual, message ) {
+		// Based on http://yuilibrary.com/yui/docs/api/files/test_js_ObjectAssert.js.html#l12.
+		var expectedKeys = YUITest.Object.keys( expected ),
+			actualKeys = YUITest.Object.keys( actual );
+
+		YUITest.Assert._increment();
+
+		// First check keys array length.
+		if ( expectedKeys.length != actualKeys.length ) {
+			YUITest.Assert.fail( YUITest.Assert._formatMessage( message,
+				'Object should have ' + expectedKeys.length + ' keys but has ' + actualKeys.length ) );
+		}
+
+		// Then check values.
+		for ( var name in expected ) {
+			if ( expected.hasOwnProperty( name ) ) {
+				if ( expected[ name ] && typeof expected[ name ] === 'object' ) {
+					areDeepEqual( expected[ name ], actual[ name ] );
```

Comment:
Same as https://github.com/ckeditor/ckeditor-dev/pull/1055/files#r144820711, couldn't it be just `bender.objectAssert.areDeepEqual( ... );`